go.mod

```diff
@@ -2,6 +2,11 @@ module github.com/argoproj/argo-workflows/v3
 
 go 1.21
 
+replace (
+	github.com/argoproj/pkg => github.com/atlassian-forks/argo-pkg v0.0.0-20240104004059-b14f36544f6d
+	github.com/argoproj/pkg/s3 => github.com/atlassian-forks/argo-pkg/s3 v0.0.0-20240104004059-b14f36544f6d
+)
+
```

Comment:
This should be removed

Reply:
Are you able to review the relevant PR in https://github.com/argoproj/pkg/pull/596 as well? The changes are needed to make this PR work

Reply:
Hey @isubasinghe, just wanted to bump here. I'd like to make both dependency changes (i.e. this one and the `cobra` one below at the same time. Could you or someone else review the above `pkg` PR?

Reply:
I don't actually have any ability to merge or review on that repo.

Reply:
This has been updated now that the relevant PR has been merged